packages/@aws-cdk-testing/cli-integ/lib/with-cdk-app.ts

```diff
@@ -620,7 +628,7 @@ export async function installNpmPackages(fixture: TestFixture, packages: Record<
   }, undefined, 2), { encoding: 'utf-8' });
 
   // Now install that `package.json` using NPM7
-  await fixture.shell(['node', require.resolve('npm'), 'install']);
+  await fixture.shell(['node', '/Users/sumughan/Developer/aws-cdk/packages/@aws-cdk-testing/cli-integ/lib/package-sources/repo-tools/npm', 'install']);
```

Comment:
you have your personal path hardcoded.

Reply:
Yup this is temporary to get the cli-integ test to work.